yb-voyager/src/tgtdb/target.go

```diff
@@ -46,7 +49,7 @@ func (t *Target) DB() *TargetDB {
 func (t *Target) GetConnectionUri() string {
 	if t.Uri == "" {
 		t.Uri = fmt.Sprintf("postgresql://%s:%s@%s:%d/%s?%s",
-			t.User, t.Password, t.Host, t.Port, t.DBName, generateSSLQueryStringIfNotExists(t))
+			t.User, url.QueryEscape(t.Password), t.Host, t.Port, t.DBName, generateSSLQueryStringIfNotExists(t))
```

Comment:
This function will check and only escapes the special characters, right?
Have tested with some different passwords? 
Maybe you can take the help of google for some different passwords - https://www.lifewire.com/strong-password-examples-2483118

Reply:
I tested it with this password "Test@123#$%^&*" -> it converts it into this - `Test%40123%23%24%25%5E%26%2A` and that worked.

Reply:
@def- we don't create users via yb-voyager. So i don't this is needed?

Reply:
You can specify an existing `--target-db-user`, so someone could have created it manually on YBDB before.

Reply:
yeah right, got it, thanks.